api/src/OpenTelemetry/Exporter/LogRecord.hs

```diff
@@ -1,2 +1,20 @@
-module OpenTelemetry.Exporter.LogRecord () where
+module OpenTelemetry.Exporter.LogRecord (
+  LogRecordExporter,
+  LogRecordExporterInternal (..),
```

Comment:
It's unfortunate that a developer has to use an constructor called `Internal` to make a `LogRecordExporter`. I don't know if this would cause confusion.